storyblok/storyblok-design-system
# feat(STR-752): Add unlocked icon to the icon list

## Pull request type

Jira Link: [STR-752](https://storyblok.atlassian.net/browse/STR-752)




## How to test this PR

1. Open the [icon list](http://localhost:6006/?path=/story/design-system-components-sbicon--all-icons)
2. Check if the `unlocked` icon is displaying properly.



## What is the new behavior?



- This PR adds a new unlocked icon to the design system
